R/03-geojob-obj.R

```diff
@@ -43,19 +45,53 @@ setMethod(f="initialize",signature="geojob",
 #' @author Jordan S Read
 #' @rdname geojob-methods
 #' @export
-setGeneric("geojob", function(...) {
+setGeneric("geojob", function(xml, ...) {
```

Comment:
The reason this ended up as xml was because there was already an xml slot that could be used, but we want using that to trigger a bit more logic.  I do think xml probably isn't the right thing to have here though.